packages/esm-outpatient-app/src/helpers/functions.ts

```diff
@@ -58,3 +58,15 @@ export function generateVisitQueueNumber(service, maxVisitNumber) {
   const visitNumber = (maxVisitNumber + 1)?.toString().padStart(4, '0');
   return servicePrefix + '-' + visitNumber;
 }
+
+export function findObsByConceptUUID(arr, id) {
```

Comment:
we could add types for `arr` and `id` to better understand what this function is doing